lib/basedriver/commands/find.js

```diff
@@ -241,12 +239,13 @@ helpers.findByImage = async function findByImage (b64Template, {
         fixImageTemplateScale, ...scale
       });
 
-      const comparedImage = await this.compareImages(MATCH_TEMPLATE_MODE, b64Screenshot, b64Template, {threshold, visualize});
-      rect = comparedImage.rect;
-      b64Matched = comparedImage.visualization;
-      score = comparedImage.score;
-
+      if (multiple) {
+        results.push(...(await this.compareImages(MATCH_TEMPLATE_MODE, b64Screenshot, b64Template, {threshold, visualize, multiple})));
```

Comment:
these lines are too long. Would it be possible to split them a bit to make the source more readable?